doc/ref_model/chapters/chapter04.md

```diff
@@ -288,7 +288,7 @@ Table 4-12 shows capabilities related to resources allocation
 | e.vim.cap.001 | Virtual Compute allocation | Yes/No | Capability to allocate virtual compute resources  to VNFC |
 | e.vim.cap.002 | Virtual Storage allocation | Yes/No | Capability to allocate virtual storage resources  to VNFC |
 | e.vim.cap.003 | Virtual Networking resources allocation | Yes/No | Capability to allocate virtual networking resources  to VNFC |
-| e.vim.cap.004 | Multi-tenant isolation | Yes/No | Capability to isolate resources between tenants |
+| e.vim.cap.004 | Multi-tenant isolation | Yes/No | Capability to isolate resources between tenants <br> **Note:** In a Kubernetes based infrastructure within one Kubernetes cluster multitenacy is provided only in resource management level. Isolation of execution environments requires separate Kubernetes clusters or worker nodes. |
```

Comment:
@CsatariGergely This sounds like this capability should either be marked no in the RA's RM compliance traceability matrix or at least be suitably qualified in that location. Including each RA's qualifiers in the the RM's table will get unmanageable.

Reply:
Yes, if the requirement is to provide hard multitenancy within one cluster, this should be marked as no for every Kubernetes based solution.

Reply:
> 
> 
> Yes, if the requirement is to provide hard multitenancy within one cluster, this should be marked as no for every Kubernetes based solution.

In that case, do you want to back this line out, and move it to the RA matrix?

Reply:
Moved to #738